packages/runtime/runtime-definitions/src/aliasing.md

```diff
@@ -0,0 +1,48 @@
+# Aliasing and root datastores
+
+## What is a root datastore? What is an aliased datastore?
+
+A root datastore is a datastore which is addressable by a custom identifier (client supplied) and is a direct child of the container runtime. A root datastore will never be garbage collected and it is a singleton with regards to the runtime it belongs to. The concept of a root datastore is considered legacy, mainly due to the previous (deprecated and removed) APIs which referred to such datastores as root. 'Aliased datastore' is the preferred current term to refer to such datastores.
```

Comment:
Maybe before "A root datastore will never be garbage collected" maybe say that objects create a graph (through handles). Objects that are not referenced through this graph get GC'd over time. And root objects are "origin' of this graph... (or something like that). Some link to GC documentation would be great from here, such that we could spend least amount of time on this topic, but allow people to learn more...

Reply:
I'd not even mention old tech, this will confuse people.

Reply:
I will remove the old tech but I don't believe this is a good place to talk too much about GC. That wasn't technically the point of the aliasing feature, it's a side effect.